cohort-cli/src/test/java/com/ibm/cohort/cli/MeasureCLITest.java

```diff
@@ -499,7 +499,7 @@ public void testZipFileKnowledgeArtifacts() throws Exception {
 		
 		Bundle emptyBundle = getBundle();
 		assertTrue( "Bundle should be empty", emptyBundle.isEmpty() );
-		mockFhirResourceRetrieval(get(urlMatching("/Condition.*")), emptyBundle);
+		mockFhirResourceRetrieval(get(urlMatching("/Condition.*&_count=500")), emptyBundle);
 		mockFhirResourceRetrieval(get(urlMatching("/Procedure.*")), emptyBundle);
```

Comment:
I'm not familiar enough with this unit test to know if it would matter,
but should the other `Resource`s also have a `&_count=500` for consistency?

Reply:
The same _count parameter is used everywhere. I updated that one check just to make sure that it was being used with the expectation that the rest would fall in line. It is valid as-is, but we certainly could add the parameter to the other mocks.